### DISCUSSION TITLE
Auto Throttle Variable

### DISCUSSION THREAD

**Discussion by Shug046** [nguyenquyhy/Flight-Tracker-StreamDeck#423] [Q&A]
Looking to be able to get the Auto Throttle variable to trigger the on and off images. I have tried this on both MSFS aircraft and PMDG. The toggle works as expected in the sim, but I can not get the feedback variable to change the image. Anyone have any ideas?

![Autothrottle](https://github.com/nguyenquyhy/Flight-Tracker-StreamDeck/assets/166310214/f6ceefa1-d75a-4201-9ddc-9ce07cbb7c9f)

**Comment by sunbeam60** [comment]
Just tried this on the MSFS2024 VisionJet and can't get AUTOTHROTTLE ACTIVE to work either.

**Reply by sunbeam60** [reply]
See https://github.com/nguyenquyhy/Flight-Tracker-StreamDeck/discussions/448 for more detail.
